admin/broadleaf-open-admin-platform/src/main/resources/open_admin_style/js/admin/components/entityForm-generatedUrl.js

```diff
@@ -86,8 +86,10 @@
 
             if (options != null && options.allowSlash) {
                 return val.replace(/ /g, BLC.systemProperty.urlFragmentSeparator).replace(/[^\w\s-_\/]/gi, '').toLowerCase() + '.' + valPostFix;
```

Comment:
Shouldn't the `if(ValPostFix)` check be up here to? Or at the end regardless? The if statement on this line also appends it onto the end.

Reply:
probably true
Slightly changed aproach of checking this...